astropy/coordinates/funcs.py

```diff
@@ -128,7 +131,9 @@ def spherical_to_cartesian(r, lat, lon):
 
     return cart.x, cart.y, cart.z
 
-
+@deprecated("6.0", alternative="get_body('sun')", 
+            warning_type=FutureWarning, 
+            message="The get_sun function is deprecated and will be removed in future versions. Use get_body('sun') instead.")
```

Comment:
I think this should be enough. Message is automatically generated usually.
```suggestion
@deprecated(since="6.0", alternative="get_body('sun')")
```

Thanks!

Reply:
Changes implemented! Thanks